src/Core/Prefab/Globe/GlobeTileBuilder.ts

```diff
@@ -0,0 +1,170 @@
+import * as THREE from 'three';
+import Coordinates from 'Core/Geographic/Coordinates';
+import Extent from 'Core/Geographic/Extent';
+import {
+    Projected,
+    ShareableExtent,
+    TileBuilder,
+    TileBuilderParams,
+} from '../TileBuilder';
+
+const PI_OV_FOUR = Math.PI / 4;
+const INV_TWO_PI = 1.0 / (Math.PI * 2);
+const axisZ = new THREE.Vector3(0, 0, 1);
+const axisY = new THREE.Vector3(0, 1, 0);
+const quatToAlignLongitude = new THREE.Quaternion();
+const quatToAlignLatitude = new THREE.Quaternion();
+const quatNormalToZ = new THREE.Quaternion();
+
+function WGS84ToOneSubY(latitude: number) {
+    return 1.0 - (0.5 - Math.log(Math.tan(
+        PI_OV_FOUR + THREE.MathUtils.degToRad(latitude) * 0.5,
+    )) * INV_TWO_PI);
+}
+
+export interface GlobeTileBuilderParams extends TileBuilderParams {
+    nbRow: number;
+    deltaUV1: number;
+    quatNormalToZ: THREE.Quaternion;
+}
+
+export class GlobeTileBuilder
+implements TileBuilder<GlobeTileBuilderParams> {
+    private _crs: string;
+    private _transform: {
+        coords: Coordinates[];
+        position: THREE.Vector3;
+        dimension: THREE.Vector2;
+    };
+
+    public computeExtraOffset?: (params: GlobeTileBuilderParams) => number;
+
+    public get crs(): string {
+        return this._crs;
+    }
+
+    public constructor(options: {
+        crs: string,
+        uvCount: number,
+    }) {
+        this._transform = {
+            coords: [
+                new Coordinates('EPSG:4326', 0, 0),
+                new Coordinates('EPSG:4326', 0, 0),
+            ],
+            position: new THREE.Vector3(),
+            dimension: new THREE.Vector2(),
+        };
+
+        this._crs = options.crs;
+        // Order crs projection on tiles
+
+        // UV: Normalized coordinates (from degree) on the entire tile
+        // EPSG:4326
+        // Offset: Float row coordinate from Pseudo mercator coordinates
+        // EPSG:3857
+        if (options.uvCount > 1) {
+            this.computeExtraOffset = GlobeTileBuilder._computeExtraOffset;
+        }
+    }
+
+    private static _computeExtraOffset(params: GlobeTileBuilderParams): number {
```

Comment:
Not entirely sure what this represents, might just be a temp storage for an unremarkable step of the computation? Matter of fact, I'm not exactly sure why nbRow is defined the way it is so that probably doesn't help...

Reply:
Ok, let's keep it like this then, I was just trying to get the knowledge you gathered while working on this in comments :)